lib/cisco_node_utils/cmd_ref/fabricpath.yaml

```diff
@@ -0,0 +1,178 @@
+# fabricpath
+---
+# Fabricpath feature is not available on N3K and N9K
+_exclude: [/N3/, /N9K/]
+
+aggregate_multicast_routes:
+  _exclude:
+    - /N5K/
+  kind: boolean
+  config_get: "show run fabricpath"
+  config_get_token: '/^fabricpath multicast aggregate\-routes$/'
+  config_set: "<state> fabricpath multicast aggregate-routes"
+  default_value: false
+  
+allocate_delay:
+  kind: int
+  config_get: "show fabricpath timers"
+  config_get_token: '/^Allocate Delay Timer\s+:\s+(\d+)/'
+  config_set: "<state> fabricpath timers allocate-delay <delay>"
+  default_value: 10
+
+auto_switch_id: 
+  kind: int
+  config_get: "show fabricpath switch-id"
+  config_get_token: '/^\*\s+(\d+).*No$/'
```

Comment:
`auto_switch_id` doesn't seem to be configurable and I see it's only used as a getter, but I'm wondering if it's really needed at all? If the cli `switch_id` config isn't used do we actually care about auto_switch_id?